eth_account/messages.py

```diff
@@ -28,7 +28,7 @@ def defunct_hash_message(
     Currently you can only specify the ``signature_version`` as following.
     version ``0x45`` (version ``E``): ``b'\\x19Ethereum Signed Message:\\n'``
     concatenated with the number of bytes in the message.
-    NOTE: This is the defualt version followed, if the signature_version is not specified.
+    .. note:: This is the defualt version followed, if the signature_version is not specified.
     version ``0x00`` (version ``0``): Sign data with intended validator (EIP 191).
```

Comment:
@Bhargavasomu can you build the docs locally and verify this renders correctly.  I'm thinking there's some whitespace needed here but not sure exactly where and how much.

Reply:
@pipermerriam I have fixed the docs, and the output is as follows.
[
![Screenshot from 2019-03-11 22-23-45](https://user-images.githubusercontent.com/15922456/54142388-92f1c700-444d-11e9-8cfd-00aec6750fe1.png)
](url)